data/data-pipeline/data_pipeline/score/score_m.py

```diff
@@ -493,22 +518,26 @@ def _health_factor(self) -> bool:
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        self.df[field_names.DIABETES_LOW_INCOME_FIELD] = (
+        self.df[field_names.DIABETES_LOW_INCOME_LOW_HIGHER_ED_FIELD] = (
             diabetes_threshold
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
-        self.df[field_names.ASTHMA_LOW_INCOME_FIELD] = (
+        self.df[field_names.ASTHMA_LOW_INCOME_LOW_HIGHER_ED_FIELD] = (
             asthma_threshold
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
-        self.df[field_names.HEART_DISEASE_LOW_INCOME_FIELD] = (
+        self.df[field_names.HEART_DISEASE_LOW_INCOME_LOW_HIGHER_ED_FIELD] = (
             heart_disease_threshold
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
-        self.df[field_names.LOW_LIFE_EXPECTANCY_LOW_INCOME_FIELD] = (
+        self.df[
+            field_names.LOW_LIFE_EXPECTANCY_LOW_INCOME_LOW_HIGHER_ED_FIELD
+        ] = (
             low_life_expectancy_threshold
             & self.df[field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES]
         )
+
+        # QQ: did not update this because it is not an eligibility factor in this model
         self.df[field_names.HEALTHY_FOOD_LOW_INCOME_FIELD] = (
```

Comment:
Yeah I'd vote we get rid of this too. I think it's old and spurious code so we might as well clean up.

Reply:
👍

Reply:
This got marked as resolved, but did you plan to delete the whole `self.df[field_names.HEALTHY_FOOD_LOW_INCOME_FIELD] = ...` block?

Reply:
hmm that's weird, i thought i had - oh well! good catch, ty